workflow/calculation/gmsim_templates/20.4.1.2/root_defaults.yaml

```diff
@@ -1,5 +1,5 @@
 hf:
-  version: 5.4.5.3_new_gnu
```

Comment:
A sim version shouldn't change HF version (for backwards compatibility). If you need a different HF version, you should create a new gm_sim version.

Reply:
Fair point. Will revert this change. Out of curiosity, do we have 5.4.5.3_new_gnu on NeSI?

Reply:
![image](https://user-images.githubusercontent.com/25143301/158721003-a049d474-6d1a-4d73-b1f5-fcf7d700a37e.png)
Yep we do.